sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -51,6 +52,8 @@
 # status enums for the status tracking table
 statuses = Enum("INPROGRESS", "SUCCEEDED", "FAILED", name="status")
 
+dependency_directions = Enum("UPSTREAM", "DOWNSTREAM", name="dependency_direction")
```

Comment:
It might be helpful to add a small comment of what exactly we mean by "upstream" and "downstream". Perhaps something like this: (Do I have this backwards?)

```suggestion
# "upstream" dependency means an instrument's processing depends on the existence of another instrument's data
# "downstream" dependency means that the instrument's data is used in another instrument's processing
dependency_directions = Enum("UPSTREAM", "DOWNSTREAM", name="dependency_direction")
```